docs/plugins.md

```diff
@@ -58,6 +58,13 @@ public class MyPlugin
         // My custom logic here
         // Find supported options below
     }
+
+    // To configure Resource
+    public OpenTelemetry.Resources.ResourceBuilder ConfigureResource(OpenTelemetry.Resources.ResourceBuilder builder)
+    {
+        // My custom logic here
```

Comment:
Based on current discussion.
I would put some more information here that it will be called independently for each signal without possibility to verify which signal is really configured.
What is more settings from this method can be overriden by ConfigureTracesOptions/ConfigureMeterProvider/ConfigureLogsOptions by calling SetResourceBuilder with new object.